extensions/security/runtime/src/main/java/io/quarkus/security/runtime/interceptor/check/RolesAllowedCheck.java

```diff
@@ -54,7 +54,7 @@ public void apply(SecurityIdentity identity, Method method, Object[] parameters)
         Set<String> roles = identity.getRoles();
         if (roles != null) {
             for (String role : allowedRoles) {
-                if (roles.contains(role)) {
+                if (roles.contains(role) || ("**".equals(role) && !identity.isAnonymous())) {
```

Comment:
So do we support `*`?

If it's what we documented before, we should probably support it.

Reply:
It was in the documentation, but didn't work. `**` is what used to work (and still works) on other places (such as `isUserInRole`), this commit just brings `@RolesAllowed` on par.

Reply:
OK, let's have @sberyozkin check that.